R/hypothesize.R

```diff
@@ -67,11 +66,19 @@ parse_params <- function(dots, x) {
   }
 
   # add in 1 - p if it's missing
-  if (length(dots[[p_ind]]) == 1) {
+  if(length(p_ind)){
```

Comment:
I see that point, but I think it's almost always a good idea to tell the user when the computer is trying to think for them.